Example/NYTPhotoViewer-Swift/ExamplePhoto.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  ExamplePhoto.swift
+//  NYTPhotoViewer
+//
+//  Created by Mark Keefe on 3/20/15.
+//  Copyright (c) 2015 The New York Times. All rights reserved.
+//
+
+import UIKit
+
+class ExamplePhoto: NSObject, NYTPhoto {
+
+    var image:UIImage?
```

Comment:
The property declarations in the Swift book, and most of the Swift code I've seen, add a space after the name and colon, before the type. Seems a little cramped here without that space.